package.json

```diff
@@ -179,13 +182,16 @@
     "csso": "^4.0.3",
     "electron": "^8.2.5",
     "electron-builder": "^22.6.0",
+    "electron-devtools-installer": "^3.0.0",
     "electron-notarize": "^0.3.0",
-    "eslint": "^6.8.0",
+    "eslint": "^7.0.0",
     "eslint-config-standard": "^14.1.1",
+    "eslint-loader": "^4.0.2",
```

Comment:
I guess the same as above: While I understand adding a vue-plugin is totally necessary, the question is why do we need to explicitly need the eslint-loader?

Reply:
The eslint-loader makes sure that everything that passes through webpack is formatted nicely. This is not strictly necessary, since the `yarn lint` also applies to the vue files now. I can remove it if you prefer. 

(BTW: the vue files have a few style warnings. Should I fix them in a new PR afterwards?)

Reply:
> BTW: the vue files have a few style warnings. Should I fix them in a new PR afterwards?

Style warnings? If it's about "not scoped styles", we can ignore them. I do not like to put styling into these files, and prefer to keep them concentrated in the LESS files, as I'm not a huge fan of this mingling of styles!

Reply:
Sorry, I meant "code style" warnings. Like wrong indent or different recommended order of template vs source etc.

Reply:
Ah alright. Yeah I wasn't able to adhere to the ESLint custom config for the rest of the app precisely because I didn't have time to integrate ESLint for the Vue files, so yeah, the style warnings are surely something to tackle!